migrations/016_change_customer_ofac_searches_add_blocked.up.sql

```diff
@@ -0,0 +1 @@
+alter table customer_ofac_searches add column blocked boolean;
```

Comment:
I can move it to customer_ofac_searches table. But it's part of v0.5 release (if I'm not mistaken).

Reply:
Can we include this column in the 006 migration?